### PR TITLE
[5025] Upgrade to rxjs@^7.0.0

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5025

[search][sdk-js] Remove Solr projects and related artifacts #4736 - https://github.com/craftercms/craftercms/issues/4736

### CODE REVIEW

File: packages/models/src/search.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+
+import { LookupTable } from "@craftercms/models";
+
+export interface resultHit {
```
Comment (rart) [inline packages/models/src/search.ts:19]:
```suggestion
export interface SearchResultHit {
```

File: packages/redux/src/epics/search.ts [hunk 2]
```diff
@@ -25,15 +25,16 @@ import {
   SEARCH,
   searchComplete
 } from '../actions/search';
+import {SearchResult} from "@craftercms/models/src/search";
```
Comment (rart) [inline packages/redux/src/epics/search.ts:28]:
```suggestion
import { SearchResult } from "@craftercms/models";
```

File: packages/models/src/search.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/*
```
Comment (rart) [inline packages/models/src/search.ts:1]: Export this file from the index

File: packages/search/README.md [hunk 4]
```diff
@@ -129,29 +129,6 @@ Map model
...
-  const query = createQuery('solr');
-  query.query = "*:*";
-  query.filterQueries = ['content-type:"/component/video"'];
-
-  search(query).subscribe((results) => {
-    // ...
-  });
-```
-
 You may alternatively use a different config by supplying the config object at the service call invoking time
```
Comment (rart) [inline packages/search/README.md:132]: This paragraph all alone doesn't make sense.

File: packages/search/README.md [hunk 5]
```diff
@@ -129,29 +129,6 @@ Map model
...
 
 ```typescript
```
Comment (rart) [inline packages/search/README.md:134]: The example(s) below are outdated to the present set of changes.

**Review by rart — CHANGES_REQUESTED** [review]

File: packages/search/README.md (hunk 4 above)
Comment (jvega190) [reply packages/search/README.md:132]: That refers to the ability to set the configuration separately (first example) or sending it as the second parameter in `search`. It feels that it's still valid.

**Pull request merged**